src/main/java/de/fraunhofer/aisec/codyze/config/CodyzeConfiguration.kt

```diff
@@ -69,3 +59,65 @@ class AnalysisMode {
     )
     var tsMode = TypestateMode.DFA
 }
+
+class MarkArgGroup {
+    var append = false
+    var matched = false
+
+    var mark: Array<File> = arrayOf(File("./"))
+
+    @Option(
+        names = ["-m", "--mark"],
+        paramLabel = "<path>",
+        description = ["Loads MARK policy files.\n\t(Default: \${DEFAULT-VALUE})"],
+        split = ","
```

Comment:
Do we continue using `,` or should we switch to system agnostic path separators?